dor/cli/upload.py

```diff
@@ -39,38 +44,55 @@ async def _run_upload(
     if not type_profiles:
         typer.echo("No profiles provided. Use --image or --text to specify profiles.")
         raise typer.Exit(1)
+
     profiles = generate_profiles(folder_path=Path(folder), type_profiles=type_profiles)
```

Comment:
Deciding not to handle this now, but I think we need to verify `folder` exists and exit gracefully. We check for this later in `run_upload_fileset`, but `generate_profiles` fails with a `FileNotFoundError` before it gets to that.